grails-app/conf/Config.groovy

```diff
@@ -132,6 +132,12 @@ grails.validateable.packages = [
         'org.pih.warehouse.shipment',
 ]
 
+org.quartz.scheduler.instanceName = 'DefaultQuartzPool'
+org.quartz.scheduler.instanceId = 'Config.groovy'
+org.quartz.scheduler.skipUpdateCheck = true
+org.quartz.threadPool.threadCount = 2
+org.quartz.threadPool.threadPriority = Thread.MIN_PRIORITY
```

Comment:
Do we actually know how this affects the quartz threads?

Reply:
- `instanceId` can be any string you like. I set it to the name of the config file to make it easier to understand from log output where the config was specified, but I can happily strike it.
- `skipUpdateCheck` means Quartz shouldn't phone home and tell you you're running an old version. We already know that, and they recommend disabling it for production instances. I believe our plugin does this for us, so it could be removed if you like
- `threadCount` affects how many threads Quartz will allocate in its internal pool. The default is 10 which is a lot for a 1- or 2-core machine.
- `threadPriority` affects how "nice" (in the POSIX sense) the Quartz threads are. I set them to lowest priority since as I understand it we use Quartz exclusively for background tasks that are not time-sensitive.

You can see Quartz picking up these settings at application startup:

```
2023-02-01 16:42:33,893 [localhost-startStop-1] INFO  core.QuartzScheduler  - Scheduler meta-data: Quartz Scheduler (v2.1.6) 'LegacyQuartzPool' with instanceId '/home/tomcat/.grails/openboxes-config.groovy'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 4 threads.
  Using job-store 'org.quartz.simpl.RAMJobStore' - which does not support persistence. and is not clustered.
```

Reply:
I would love to see these as comments in the configuration if possible.